packages/@aws-cdk/aws-cognito/test/integ.user-pool-resource-server.ts

```diff
@@ -0,0 +1,20 @@
+import { App, Stack } from '@aws-cdk/core';
+import { UserPool } from '../lib';
+
+const app = new App();
+const stack = new Stack(app, 'integ-user-pool-resource-server');
+
```

Comment:
Would be nice to set up a resource server + an app client that work together and document the set of steps to validate it.

The steps can be documented as so - https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md#integration-tests